### PR TITLE
feat: add falling/jump animation

### LINKED ISSUE #184: Add `jump`/`falling` animation

### PR DESCRIPTION
**Pull request by leanmendoza**
Fix #184 

Todo
- [x] test emotes
- [x] tune 
- [x] create issue to add sprint to mobile (https://github.com/decentraland/godot-explorer/issues/213)
- [x] fix test avatar attach

### CODE REVIEW

File: godot/src/decentraland_components/avatar.gd [hunk 1]
```diff
@@ -158,11 +160,15 @@ func _load_default_emotes():
...
-		animation_player.stop()
-		animation_player.play(emote_id)
+		animation_emote_node.animation = emote_id
+
+		var pb: AnimationNodeStateMachinePlayback = animation_tree.get("parameters/playback")
+		if pb.get_current_node() == "Emote":
+			pb.start("Emote", true)
+		playing_emote = true
 	else:
+		prints(animation_player.get_animation_list())
```
Comment (leanmendoza) [inline godot/src/decentraland_components/avatar.gd:170]: remove

File: godot/src/logic/player/player.gd [hunk 2]
```diff
@@ -88,10 +85,11 @@ func _on_player_profile_changed(new_profile: Dictionary):
 
 
 func _on_param_changed(_param):
-	walk_speed = Global.config.walk_velocity
-	run_speed = Global.config.run_velocity
-	gravity = Global.config.gravity
-	jump_velocity_0 = Global.config.jump_velocity
+	pass
+	#walk_speed = Global.config.walk_velocity
```
Comment (leanmendoza) [inline godot/src/logic/player/player.gd:89]: add todo

File: godot/src/test/testing_api.gd [hunk 3]
```diff
@@ -188,21 +185,29 @@ func async_take_and_compare_snapshot(
...
+	Global.scene_runner.player_node.avatar.hide()
 
 	await get_tree().process_frame
 	await get_tree().process_frame
 	await get_tree().process_frame
 
 	var viewport_img := viewport.get_texture().get_image()
 
-	#await get_tree().create_timer(10.0).timeout
+	# Test: Uncomment this to see how the snapshot would look like
```
Comment (kuruk-mm) [inline godot/src/test/testing_api.gd:206]: can we execute this behind an arg?

**Review by kuruk-mm — APPROVED** [review]
Code looks great! Just the comments that you already add the 'TODO'

File: godot/src/test/testing_api.gd (hunk 3 above)
Comment (leanmendoza) [reply godot/src/test/testing_api.gd:206]: It's not worth it

**Pull request merged**